systers_portal/meetup/tests/test_views.py

```diff
@@ -239,6 +239,13 @@ def setUp(self):
                                              meetup_location=self.meetup_location,
                                              created_by=self.systers_user,
                                              last_updated=timezone.now())
+        self.meetup3 = Meetup.objects.create(title='Foo Baz', slug='foobar',
```

Comment:
I have made 3 meetups in all, one which is upcoming (self.meetup2), one which is past (self.meetup3) and one whose date is the current date (self.meetup). Only the meetup which has passed should show up in 'past meetups'.